staff/pedro-garriganogues/ecomerce/style.css

```diff
@@ -0,0 +1,227 @@
+html, body {
+    height: 100%;
+}
+
+#header {
+    border: 1px solid black;
+    width: 100%;
+    height: 20%;
+    box-sizing: border-box;
+    position: fixed;
+    padding: 10px;
+    background-image: url(https://c.wallhere.com/photos/f0/61/california_road_trip_travel_blue_nature_sign_back-555704.jpg!d);
+   
+}
+
+div.desc {
```

Comment:
Here you con't need to put div.desc, only writing the class name is enough, so .desc